trie/leafNode.go

```diff
@@ -144,11 +144,11 @@ func (ln *leafNode) tryGet(key []byte, currentDepth uint32, _ common.TrieStorage
 	return nil, currentDepth, nil
 }
 
-func (ln *leafNode) getNext(key []byte, _ common.TrieStorageInteractor) (node, []byte, error) {
+func (ln *leafNode) getNext(key []byte, _ common.TrieStorageInteractor) (node, []byte, []byte, error) {
```

Comment:
it would be better to have a structure to be returned instead of node, []byte, []byte

Reply:
Done